iwyu_output.cc

```diff
@@ -475,7 +477,6 @@ string OneIncludeOrForwardDeclareLine::LineNumberString() const {
   return buf;
 }
 
-
```

Comment:
No idea why it was there -- also no idea why it was removed.  I probably thought I accidentally added it and so removed it, and then missed it in the diff.  I'll make sure my diff tool isn't set to ignore whitespace. which I have to set it to from time to time. I can add it back.